Src/BootCamp.Chapter/Player.cs

```diff
@@ -0,0 +1,138 @@
+﻿using BootCamp.Chapter.Items;
+using System;
+using System.Net.NetworkInformation;
+using System.Reflection.Metadata;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5776099)